success-criteria.md

```diff
@@ -113,34 +113,19 @@ behind this decision ought to be thoroughly documented.
 
 This document considers web advertising from three [points of
 view](https://www.w3.org/Consortium/Points/)[POINTS]: that of individual web
-users (both in aggregate and individually), publishers and their partners (both
+users (both in aggregate and individually), --publishers and their partners (both
 authors and the business model that funds them) and the delivery access
-mechanism (both connectivity and navigation).
-
-Any technology can be abused. Open societies do not attempt to suppress
-technology, but rather put [appropriate
-regulations](https://iabeurope.eu/all-news/iab-europes-press-statement-openrtb-and-eu-data-protection-law)
-[OPEN-RTB-EU-LAW] in place to define acceptable and unacceptable uses of that
-technology. For example, automobiles are not required to integrate functionality
-that technically prevents them from exceeding the speed limit. Instead, drivers
-are educated and trained in traffic rules, and drivers who violate speed limits
-are subject to fines and/or deprived of their permits. However, documenting
-specific criteria as to what constitutes a violation helps enable easier
-detection and reporting of non-compliance with the regulations that govern
-technology.
+mechanism (both connectivity and navigation).--
+
+Any technology can be abused. Any technology can be abused. Open societies consider all technologies but define ethical and unethical uses and regulate or restrict accordingly. Documenting specific criteria as to what constitutes an ethical use of a technology helps enable easier detection and reporting of non-compliance with the regulations that govern societies and support the process of constructing such regulations.
```

Comment:
Open functioning societies suppress technology all the time, electric chairs, polluting exhaust systems, microbeads, etc... just to name a few. While you could argue that these are more general tech categories on which societies apply regulations, I would say that this - in a statement document that I presume wants publisher support - is over-extensive in its descriptive scope and counter some of the public arguments that publishers have made in regard to specific web technologies. Indeed, some look towards the active end of 3p as a suppression of an incorrectly adopted technology. Marking for removal. Suggested replacement to handle these concerns and focus in on the societal impact that concerns us, along with the specific focus we should have as a group.

Reply:
@darobin I'm aligned on your statements related to the need to define acceptable data processing and that people should not have to choose between privacy and access to quality journalism.

We also agree that advertising funds the majority of people's access to web content, including quality journalisim. Accordingly we highlighted this access as an important success criteria for any proposal.

As we improve the web to make it more user friendly and publisher friendly, we need to ensure that marketers funding of people's access is not undermined whilst respecting people's trust choices.

In relation to improving web advertising this document does outline some requirements concering data processing by publishers, advertisers and their supply chain. I look forward to reading your draft document on acceptable data processing when it is ready.